orttraining/orttraining/core/framework/module_gradient_graph_builder.cc

```diff
@@ -77,8 +77,13 @@ Status ModuleGradientGraphBuilder::Build(const std::vector<std::vector<int64_t>>
     SetConcreteInputShapes(*input_shapes_ptr);
   }
 
-  // Build the gradient graph.
-  ORT_RETURN_IF_ERROR(BuildGradientGraph());
+  // Optimize the inference graph, and if needed, build the gradient graph.
```

Comment:
Should we rename this class from `ModuleGradientGraphBuilder` to `ModuleGraphBuilder`?

Reply:
Maybe rename from `orttraining/orttraining/core/framework/module_gradient_graph_builder.cc` to `orttraining/orttraining/core/framework/module_graph_builder.cc`

Reply:
Done